src/services/api/AxiosInstance.ts

```diff
@@ -10,10 +11,10 @@ import tracer from "@utils/tracer"
 // Env vars
 const GITHUB_ORG_NAME = config.get("github.orgName")
 
-const requestFormatter = async (config: AxiosRequestConfig) => {
+const requestFormatter = async (axiosConfig: AxiosRequestConfig) => {
   logger.info("Making GitHub API call")
 
-  const authMessage = config.headers.Authorization
+  const authMessage = axiosConfig.headers?.Authorization
```

Comment:
no, because we use this to determine if they are email user. subsequent lines below: 
```ts
const isEmailLoginUser =
  !authMessage ||
  authMessage === "token " ||
  authMessage === "token undefined"

if (isEmailLoginUser) {
  const accessToken = await getAccessToken() 
  // do some work
} 
```